spec/requests/session_controller_spec.rb

```diff
@@ -3009,6 +3010,146 @@ def post_login
     end
   end
 
+  describe "#passkey_challenge" do
+    it "returns a challenge for an anonymous user" do
```

Comment:
Just to confirm, is this used when creating a new user?

Reply:
O wait no this is for when we want to login a new user and hence why we need to allow the endpoint for anon.